streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java

```diff
@@ -64,6 +64,10 @@
 
         // initialize the topology with its own context
         processorContext = new StandbyContextImpl(id, applicationId, config, stateMgr, metrics);
+
+        final LogContext logContext = new LogContext(logPrefix);
```

Comment:
We already perform these 2 lines in `AbstractTask` do we need to do it again?

Reply:
Fixed